kubernetes/loculus/values.yaml

```diff
@@ -1719,7 +1719,7 @@ resources:
   ingest:
     requests:
       memory: "1Gi"
-      cpu: "200m"
+      cpu: "20m"
```

Comment:
This only makes sense to me as a change specifically for previews?

Reply:
(same with the others below)

Reply:
All these values here are for previews, for production and PPX we use different ones and there's no real problem in setting these requests on the lower end as in the worst case it'll cause throttling which isn't a problem for ingest

Reply:
Yes but it will affect e.g. CovSpectrum and anyone else who tries out Loculus. In this values file we should aim (mostly) for sensible defaults for a generic production environment. I would think this is a bit low for that

Reply:
but ok, I see LAPIS below isn't that on main

Reply:
or actually maybe LAPIS doesn't need much- I guess SILO does the heavy lifting